content/data/settings.json

```diff
@@ -33,5 +33,20 @@
     "mediaAltAtts": true,
     "linksAltAtts": true,
     "singularTagLinks": true
-  }
+  },
+  "embeddable_view": {
+    "host": "https://bafybeibjasmf2bvswm3ejg4shppxtw5l36bfn7uouypjm6teqwpcpxi4pi.on.fleek.co/",
```

Comment:
@orvn  I missed this one, do we need to change this to `ecosystem.ipfs.tech`?

Reply:
I'm around if you want to push this over the finish line and add this section to https://ipfs.tech